lib/src/model/field.dart

```diff
@@ -92,6 +92,12 @@ class Field extends ModelElement
     return field.isFinal;
   }
 
+  @override
+  bool get isLate {
+    if (isFinal) return field.isLate;
```

Comment:
Consider `return isFinal && field.isLate;`

Reply:
changed to bool get isLate => isFinal && field.isLate;